plugins/woocommerce/src/Admin/API/Reports/DataStore.php

```diff
@@ -48,9 +48,16 @@
  * </code></pre>
  *
  * Please use the `woocommerce_data_stores` filter to add your custom data store to the list of available ones.
- * Then, your store could be accessed by Query classes ({@see GenericQuery::get_data() GenericQuery::get_data()})
+ * Then, your store could be accessed by Controller classes ({@see GenericController::get_datastore_data() GenericController::get_datastore_data()})
  * or using {@link \WC_Data_Store::load() \WC_Data_Store::load()}.
```

Comment:
Checking the generated report and I found the `@see` and `@link` here were not being transformed to the actual link. Would this related to the https://github.com/woocommerce/woocommerce/pull/50748#discussion_r1723766187 you added that we might need to fix the code reference?

<img width="957" alt="Screenshot 2024-08-21 at 12 18 51" src="https://github.com/user-attachments/assets/b1069cb1-979b-4c44-a710-a324c013bb83">

Reply:
yes, exactly. I didn't manage the code reference to work with them. I think we can fix that later.